.github/workflows/linux.yml

```diff
@@ -59,7 +62,13 @@ jobs:
           QT_PKGS: ${{ matrix.qt-version-major == '6' && 'qt6-base-private-dev libgl-dev' || 'qtbase5-private-dev' }}
         run: |
           sudo apt update
-          sudo apt install -y build-essential qtchooser ${{ env.QT_PKGS }}
+          sudo apt install -y build-essential qtchooser ${{ env.QT_PKGS }} cmake ninja-build
```

Comment:
Please remove `qtchooser`, it was only necessary for `qmake`. Also, can you please sort the package names alphabetically? 🥺 Thank you ❤️